backend/src/main/java/com/woowacourse/kkogkkog/domain/CouponStatus.java

```diff
@@ -41,4 +45,11 @@ private CouponStatus handleDecline() {
         }
         return READY;
     }
+
+    private CouponStatus handleAccept() {
+        if (this != REQUESTED) {
+            throw new InvalidRequestException("사용 요청을 수락할 수 없는 상태의 쿠폰입니다.");
+        }
```

Comment:
이 친구만 예외적으로 이런 문구로 가는건 어떨까요?

```suggestion
        if (this != REQUESTED) {
            throw new InvalidRequestException("사용 요청 상태의 쿠폰이 아닙니다.");
        }
```

Reply:
좋습니다 반영했습니다